packages/ag-charts-enterprise/src/features/annotations/scenes/lineWithTextScene.ts

```diff
@@ -102,12 +102,14 @@ export class LineWithTextScene {
         let [left, right] = Vec2.from(coords);
         if (left.x > right.x) [left, right] = [right, left];
 
-        const fontOffset = (fontSize ?? 14) / 3;
-
         const normal = Vec2.normalized(Vec2.sub(right, left));
         const angle = Vec2.angle(normal);
-        const inset = Vec2.multiply(normal, DivariantHandle.HANDLE_SIZE / 2 + fontOffset);
-        const offset = Vec2.multiply(normal, (strokeWidth ?? 2) / 2 + fontOffset);
+
+        // Inset from the end of the line
+        const inset = Vec2.multiply(normal, DivariantHandle.HANDLE_SIZE / 2 + (fontSize ?? 14) / 2);
```

Comment:
This changes the `fontOffset` from `(fontSize ?? 14) / 3` to `(fontSize ?? 14) / 2`.
Can you confirm that this is intentional?

Reply:
Yep, a bit of tweaking of values from design